tests/page/page-drag.spec.ts

```diff
@@ -19,8 +19,8 @@ import { test as it, expect } from './pageTest';
 import { attachFrame } from '../config/utils';
 
 it.describe('Drag and drop', () => {
-  it.skip(({ isAndroid }) => isAndroid);
-  it.fixme(({ browserName }) => browserName === 'chromium');
+  it.skip(({isAndroid}) => isAndroid);
+  it.skip(({isChromium, browserMajorVersion}) => isChromium && browserMajorVersion < 91);
```

Comment:
I was under impression that we are waiting for the stable to pick up the protocol changes.

Reply:
There is a try-catch in the code.